recipes/omniscidb-cuda/recipe/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "omniscidb-cuda" %}
+{% set version = "4.8.1" %}
+{% set sha256 = "300cdbbaa9108879f7ec22d9cf6210ca2700a3ef99c356a5c00fbf3aeb3a4713" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/omnisci/omniscidb/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    Skip-installing-Boost-libs.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    #- {{ compiler('c') }}
+    - boost-cpp 1.68.*        # to avoid cmake picking up boost-cpp 1.70+
+    - cmake
```

Comment:
Could you please elaborate on this `boost-cpp`/`cmake` workaround?

Reply:
omniscidb is not `boost > 1.68` ready. Currently, when using 1.70, the build fails with
```
undefined reference to `boost::program_options::options_description::m_default_line_length'
```

I'll get back to here when I reproduce the issue that required `boost-cpp` in `build` section.

Reply:
> omniscidb is not `boost > 1.68` ready. Currently, when using 1.70, the build fails with
> 
> ```
> undefined reference to `boost::program_options::options_description::m_default_line_length'
> ```

That's fine.

> I'll get back to here when I reproduce the issue that required `boost-cpp` in `build` section.

This is what I'm interested in understanding. Guessing this is a bug with something else, but understand what needs to be fixed would be useful.

Reply:
The `boost-cpp/cmake` workaround appears to be unnecessary now.